projects/igniteui-angular/src/lib/accordion/accordion.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="igx-accordion__root" role="accordion" (keydown)="handleKeydown($event)">
```

Comment:
Not sure if this wrapper is needed. It could be
```typescript
/**
 * The role attribute of the accordion.
**/
@HostBinding('attr.role')
public role="accordion";
...
/** @hidden @internal **/
@HostBinding('class.igx-accordion__root')
public cssClass = "igx-accordion__root"
...
/** @hidden @internal **/
@HostListener('keydown', [$event])
public handleKeydown(event: KeyboardEvent) {
   ...
}
```

Reply:
This can be discussed w/ the design team, once (if) styles for the accordion are being implemented